src/kubernetes_configmap_jwt_scan/kubernetes_configmap_jwt_scan/scan.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+import logging
+import os
+import re
+import shutil
+import tarfile
+import jwt
+
+from distutils.dir_util import mkpath
+
+import kubernetes_configmap_jwt_scan.context
+
+from kubernetes_configmap_jwt_scan.config import config
+from lib.kctl import Kctl
+
+WARN_THRESHOLD = 30 # warn if expiring within 30 days
+JWT_REGEX = re.compile('^(?:[\w-]*\.){2}[\w-]*$')
+
+def is_jwt(str):
```

Comment:
Wonder if this function can go into src/lib and a test be written for it.

Reply:
Done in https://github.com/artsy/opstools/pull/45/commits/4c6976bbc6f2fefb6810885e151cac24fb59ad8c